applications/notify/src/notify_low_balance.erl

```diff
@@ -77,7 +77,7 @@ send(CurrentBalance, Account) ->
 -spec create_template_props(integer(), kz_json:object()) -> kz_proplist().
 create_template_props(CurrentBalance, Account) ->
     AccountDb = kz_util:format_account_id(kz_doc:id(Account), 'encoded'),
-    Threshold = notify_account_crawler:low_balance_threshold(AccountDb),
+    Threshold = ananke_account_crawler:low_balance_threshold(AccountDb),
```

Comment:
@icehess you should not use functions from another application ;)  Perhaps this should be in core or in kazoo_documents...